configuration/rules-blockly-items-things.md

```diff
@@ -0,0 +1,181 @@
+---
+layout: documentation
+title: Rules Blockly - Items & Things
+---
+
+# Items &  Things
+[return to Blockly Reference]({{base}}/configuration/rules-blockly.html)
+
+*Items* and *Things* are the [major entities of openHAB](https://www.openhab.org/docs/concepts/) to control and monitor the home. These can be accessed via the "Items & Things" section of the [Blockly Toolbox]({{base}}/configuration/rules-blockly.html#blockly-toolbox).
```

Comment:
Please change this to one sentence per line.
The website build will take core of a proper presentation of paragraphs.